ci_framework/roles/edpm_deploy_baremetal/defaults/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+# All variables within this role should have a prefix of "cifmw_edpm_deploy_baremetal"
+# cifmw_edpm_deploy_baremetal_basedir: "{{ cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data') }}"
+cifmw_edpm_deploy_baremetal_manifests_dir: "{{ cifmw_manifests | default(cifmw_edpm_deploy_baremetal_basedir ~ '/artifacts/manifests') }}"
+cifmw_edpm_deploy_baremetal_dry_run: false
+cifmw_edpm_deploy_baremetal_skip_crc_storage_creation: false
+cifmw_install_yamls_defaults:
```

Comment:
@pablintino care to double-check that usage? usually we would get a `cifmw_edpm_deploy_baremetal: "{{ cifmw_install_yamls_defaults | default({'NAMESPACE': 'openstack'}) }}` I think, since iirc the NAMESPACE is dynamically loaded and the var is provisioned at some early step.

Reply:
Seems to be good, we are pointing in the install_yamls role the `OUT` var to that directory, ` ~/ci-framework-data/artifacts/manifests`, in where install_yamls itself creates the correct structure based on namespaces, now two, the operators one and the plain openstack one for services. Anyway, is this variable even used? Based on the use case, if we are not going to read/create a manifest shouldn't be necessary to pass anything, as the `OUT` variable, is for now added in the legacy `edpm_env`.